pandas/core/indexes/multi.py

```diff
@@ -2597,9 +2596,7 @@ def _get_partial_string_timestamp_match_key(self, key):
         return key
 
     @Appender(_index_shared_docs["get_indexer"] % _index_doc_kwargs)
-    def get_indexer(self, target, method=None, limit=None, tolerance=None):
-        method = missing.clean_reindex_fill_method(method)
-        target = ensure_index(target)
+    def _get_indexer(self, target: Index, method=None, limit=None, tolerance=None):
```

Comment:
i think can remove the docs on anything you renamed to _get_indexer

Reply:
sure.  will do this in the on-deck follow-on